### PR TITLE
Update usage and docs

### LINKED ISSUE #68: Update documentation with recommendations for distributed file systems
### Description of feature

When running the alphafold2 mode the DBs are too big to be staged in standard file systems and thus, distributed file systems such as FSx are needed, see for instance [here](https://github.com/aws-samples/aws-batch-arch-for-protein-folding#5-architecture-details). For this reason, the documentation should be updated to reflect these requirements.

### PR DESCRIPTION
**Pull request by athbaltzis**
- Add output images
- Add metro map image for the pipeline
- Update README and usage documents
- Add recommendation for the use of shared file systems (closes #68)

## PR checklist

- [X] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/proteinfold/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/proteinfold _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [X] Make sure your code lints (`nf-core lint`).
- [X] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [X] Usage Documentation in `docs/usage.md` is updated.
- [X] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [X] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -128,11 +128,11 @@ The nf-core/proteinfold pipeline comes with documentation about the pipeline [us
 
 ## Credits
 
-nf-core/proteinfold was originally written by Athanasios Baltzis, Jose Espinosa-Carrasco, Luisa Santus, Harshil Patel.
+nf-core/proteinfold was originally written by Athanasios Baltzis ([@athbaltzis](https://github.com/athbaltzis)), Jose Espinosa-Carrasco ([@JoseEspinosa](https://github.com/JoseEspinosa)), Luisa Santus ([@luisas](https://github.com/luisas)), Harshil Patel ([@drpatelh](https://github.com/drpatelh)).
```
Comment (JoseEspinosa) [inline README.md:131]:
```suggestion
nf-core/proteinfold was originally written by Athanasios Baltzis ([@athbaltzis](https://github.com/athbaltzis)), Jose Espinosa-Carrasco ([@JoseEspinosa](https://github.com/JoseEspinosa)) and Luisa Santus ([@luisas] from [The Comparative Bioinformatics Group](https://www.crg.eu/en/cedric_notredame) at [The Centre for Genomic Regulation, Spain](https://www.crg.eu/) under the umbrella of the [BovReg project](https://www.bovreg.eu/) (https://github.com/luisas)) and Harshil Patel ([@drpatelh](https://github.com/drpatelh)) Harshil Patel ([@drpatelh](https://github.com/drpatelh)) from [Seqera Labs, Spain](https://seqera.io/).
```

File: README.md [hunk 2]
```diff
@@ -128,11 +128,11 @@ The nf-core/proteinfold pipeline comes with documentation about the pipeline [us
...
 
 We thank the following people for their extensive assistance in the development of this pipeline:
 
-<!-- TODO nf-core: If applicable, make list of people who have also contributed -->
+[@martin-steinegger](https://github.com/martin-steinegger), [@rjpbonnal](https://github.com/rjpbonnal), [@l-mansouri](https://github.com/l-mansouri)
```
Comment (JoseEspinosa) [inline README.md:135]:
```suggestion
Many thanks to others who have helped out and contributed along the way too, including (but not limited to): [@martin-steinegger](https://github.com/martin-steinegger), [@rjpbonnal](https://github.com/rjpbonnal) and [@l-mansouri](https://github.com/l-mansouri)
```

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]
Looks good only I suggested some changes in the text

File: README.md (hunk 1 above)
Comment (athbaltzis) [reply README.md:131]: Great! Can you just move "(https://github.com/luisas))" after Luisa's name?

File: README.md (hunk 2 above)
Comment (athbaltzis) [reply README.md:135]: I will add with some more in a commit that I am preparing

**Review by JoseEspinosa — APPROVED** [review]
Awesome!

**Pull request merged**